project.envsubst.yml

```
@@ -16,6 +15,10 @@ settings:
     MARKETING_VERSION: 1.0
     CURRENT_PROJECT_VERSION: 1
     ENABLE_USER_SCRIPT_SANDBOXING: "NO"
+    DEVELOPMENT_TEAM: $TEAM_ID
+  configs:
+    Release:
+      SWIFT_OPTIMIZATION_LEVEL: "-O" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)